cirq/protocols/resolve_parameters.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, TypeVar
+from typing_extensions import Protocol
+from cirq.study import ParamResolver
+
+TDefault = TypeVar('TDefault')
+
+class SupportsParameterization(Protocol):
+    """An object that may be describable by a unitary matrix."""
```

Comment:
Incorrect doc.

Reply:
Uh oh, it appears he has discovered my main programming tactic (cut and paste).
Fixed.